terraform/ecs/cluster.tf

```
@@ -86,6 +86,8 @@ resource "aws_ecs_task_definition" "app_task" {
         { name = "RPC_PROXY_PROVIDER_COINBASE_API_KEY", value = var.coinbase_api_key },
         { name = "RPC_PROXY_PROVIDER_COINBASE_APP_ID", value = var.coinbase_app_id },
         { name = "RPC_PROXY_PROVIDER_ONE_INCH_API_KEY", value = var.one_inch_api_key },
+        { name = "RPC_PROXY_PROVIDER_GETBLOCK_ACCESS_TOKENS_JSON", value = var.getblock_api_tokens },
```

Would be good to keep the names consistent

```suggestion
        { name = "RPC_PROXY_PROVIDER_GETBLOCK_ACCESS_TOKENS_JSON", value = var.getblock_access_tokens },
```